packages/admin/admin/src/alerts/AdminComponentAlerts.tsx

```diff
@@ -0,0 +1,156 @@
+    | "closeIconFull"
+    | "marginBottom"
+    | "paddingLeft"
+    | "cursorPointer";
+
+const styles = (theme: Theme) =>
+    createStyles<AdminComponentAlertsClassKey, AdminComponentAlertsProps>({
+        root: {
+            backgroundColor: theme.palette.background.paper,
+            borderRadius: 4,
+            width: 400,
+            padding: 20,
+            display: "flex",
+            boxShadow: "0px 0px 20px 0px rgba(0, 0, 0, 0.10)",
+            position: "relative",
+        },
+        content: {},
+        title: {
+            fontWeight: "bold",
+        },
+        text: {
+            fontWeight: 300,
+            fontSize: 14,
+            whiteSpace: "nowrap",
+        },
+        button: {
+            paddingLeft: 0,
+        },
+        container: {
+            display: "flex",
+            alignItems: "center",
+        },
+        closeIconFull: {
+            position: "absolute",
+            right: 10,
+            top: 10,
+        },
+        marginBottom: {
+            marginBottom: 10,
+        },
+        cursorPointer: {
+            cursor: "pointer",
+        },
+        paddingLeft: {
+            paddingLeft: 180,
+            paddingRight: 10,
+        },
+        severityInfo: {
+            border: `1px solid ${theme.palette.primary.main}`,
+            borderLeft: `5px solid ${theme.palette.primary.main}`,
+        },
+        severityWarning: {
+            border: `1px solid ${theme.palette.warning.main}`,
+            borderLeft: `5px solid ${theme.palette.warning.main}`,
+        },
+        severityError: {
+            border: `1px solid #D11700`,
+            borderLeft: `5px solid #D11700`,
+        },
+        severitySuccess: {
+            border: `1px solid ${theme.palette.secondary.main}`,
+            borderLeft: `5px solid ${theme.palette.secondary.main}`,
+        },
+    });
+
+function AdminComponentAlerts({
+    severity = "info",
+    startIcon,
+    title,
+    text,
+    classes,
+    hideCloseButton = false,
+    onCloseClick,
+}: AdminComponentAlertsProps & WithStyles<typeof styles>): React.ReactElement {
+    return (
+        <Box
+            className={clsx(
+                classes.root,
+                !title && classes.container,
+                severity === "info" && classes.severityInfo,
+                severity === "warning" && classes.severityWarning,
+                severity === "error" && classes.severityError,
+                severity === "success" && classes.severitySuccess,
+            )}
+        >
+            {severity === "info" ? (
+                <Info color={severity} />
+            ) : severity === "warning" ? (
+                <Warning color={severity} />
+            ) : severity === "error" ? (
+                <Error color={severity} />
+            ) : severity === "success" ? (
+                <Check color={severity} />
+            ) : null}
+
+            <Container className={clsx(!title && classes.container)}>
```

Comment:
I don't think the `container` class should be dependent on the `title`... it's still the `container` when no title is set.